python/main.py

```diff
@@ -9,24 +9,30 @@
 
 with open(temp_file, 'r') as f:
     print(f)
+    
 
 os.chmod("/tmp/foo.txt", stat.S_IXGRP)
 tar_file = '/file.tax*'
 os.system(tar_file)
 
+
 KEY_SIZE = 1024
+
```

Comment:
SCA

<h2>MOCK_PACKAGE_NAME 1.1.2 / ec2.tf</h2>
<h3>Total vulnerabilities: 2</h3>

<table>
                    <tr>
                        <th>Critical: 1</th>
                        <th>High: 1</th>
                        <th>Medium: 0</th>
                        <th>Low: 0</th>
                    </tr>
                </table>

<table>
<tr><th>Vulnerability ID</th><th>Severity</th><th>CVSS</th><th>Fixed in</th><th>Status</th></tr>
<tr>
                                <td><a href="https://www.bridgecrew.cloud/projects?repository=bridgecrew/platform&branch=15513_client-workspace-phase-1&repoUUID=d7d49366-58e0-4b72-9f62-e26bf0ff5879&runId=latest&viewId=VCSPullRequests">CVE-2022-1650</a></td>
                                <td><img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsSeverityIcons/critical_new.svg" alt="CRITICAL" width="24px" align="center">&nbsp;CRITICAL</td>
                                <td>9.3</td>
                                <td><code>1.1.1</code></td>
                                <td>Open</td>
                            </tr>
<tr>
                                <td><a href="https://www.bridgecrew.cloud/projects?repository=bridgecrew/platform&branch=15513_client-workspace-phase-1&repoUUID=d7d49366-58e0-4b72-9f62-e26bf0ff5879&runId=latest&viewId=VCSPullRequests">CVE-2020-7788</a></td>
                                <td><img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsSeverityIcons/high_new.svg" alt="HIGH" width="24px" align="center">&nbsp;HIGH</td>
                                <td>7.3</td>
                                <td><code>1.0.5</code></td>
                                <td>Open</td>
                            </tr>
</table>
<p><img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsBotLogos/bc.svg" alt="Bridgecrew" width="18px">&nbsp;<small>Comment by Bridgecrew Bot</small></p>